.github/workflows/test.yaml

```diff
@@ -38,10 +39,71 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
-        if: always()
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Save Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Pull a small test Docker image.
+        run: docker pull ghcr.io/linuxserver/baseimage-alpine:edge-version-f502fcfd
+  restore-cache:
+    name: Restore Cache
+    needs:
+      - save-cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Load Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Verify Docker loaded empty image from cache.
+        run: |
+          description="$(docker inspect --format '{{ index .Config.Labels "description" }}' empty)"
+          [[ $description == "empty image" ]]
```

Comment:
This test and the step name need to be updated to reflect the fact that we are no longer building a Docker image.

```yaml
title="$(docker inspect --format '{{ index .Config.Labels "org.opencontainers.image.title" }}' ghcr.io/linuxserver/baseimage-alpine:edge-version-f502fcfd)"
[[ $title == "Baseimage-alpine" ]]
```